server/src/main/java/org/elasticsearch/transport/OutboundHandler.java

```diff
@@ -218,7 +170,8 @@ private void maybeLogSlowMessage() {
         protected void innerOnResponse(Void v) {
             assert messageSize != -1 : "If onResponse is being called, the message should have been serialized";
```

Comment:
`messageSize` is always nonnegative now right?

Reply:
yea right this can go away :)